invenio_app_ils/records_relations/api.py

```diff
@@ -109,11 +110,11 @@ def _build_relations_object(self, pid_value, pid_type, **kwargs):
         r.update(kwargs)
         return r
 
-    def _build_parent(self, child_pid, relation_type):
+    def _build_parent(self, parent_pid, relation_type):
```

Comment:
actually it looks to me that this `_build_parent` is never called. If true, you can remove it.